docs/docs/playground.mdx

```diff
@@ -82,7 +82,10 @@ The following playground uses most of the provided plugins.
         searchHighlightPlugin,
       ];
 
-      plugins.push(createDeserializeHTMLPlugin({ plugins }));
+      plugins.push(...[
+        createDeserializeHTMLPlugin({ plugins }),
```

Comment:
For fun, might want to add markdown and csv as well... order would be csv, md, html, ast

Reply:
Had a typo, md before csv since the order is reversed.